Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -184,19 +189,40 @@ - (void)setSelectionType:(BPKCalendarSelection)selectionType {
     return self.calendarView.selectedDates;
 }
 
+- (NSSet<BPKSimpleDate *> *)createDateSet:(NSArray<NSDAte *> *)dates {
```

Comment:
`NSDate`

Reply:
Strange that it even compiles

Reply:
I didn't I refactored without recompiling 😂

Reply:
Yeah - it didn't compile.
Wait, you made all these changes without testing them, and they work? You really are some sort of programming legend.

Reply:
😮